client/app/tests/phpunit/Controller/UserControllerTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
The same comments have been made on `CoDeputyControllerTest`

Reply:
There were no tests in place on the client side, so I am creating a couple of tickets to get them created/refactored to be proper unit tests.